packages/svelte/src/internal/client/runtime.js

```diff
@@ -1458,16 +1458,15 @@ export function is_store(val) {
 export function prop(props, key, flags, initial) {
 	var immutable = (flags & PROPS_IS_IMMUTABLE) !== 0;
 	var runes = (flags & PROPS_IS_RUNES) !== 0;
-
-	var setter = get_descriptor(props, key)?.set;
-	if (DEV && setter && runes && initial !== undefined) {
-		// TODO consolidate all these random runtime errors
-		throw new Error('Cannot use fallback values with bind:');
-	}
-
 	var prop_value = /** @type {V} */ (props[key]);
+	var setter = get_descriptor(props, key)?.set;
 
 	if (prop_value === undefined && initial !== undefined) {
+		if (DEV && setter && runes) {
```

Comment:
This shouldn't be a DEV time only error, like others it should have a short message. In doesn't make sense that this would error only in DEV and not prod. Not sure why the previous could from before was also DEV too FWIW.